File: AreaOfCircle/Area.java
/*
    Author : Pakshal Jain
    Date : 28/10/2022
    Program : To Accept Radius From User and Calculate Area of Circle
*/

//Required Libraries
import java.lang.*;
import java.util.*;

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Class Declaration
class Calculate 
{
    //Characteristics of Class
    float No = 0.0f;
    
    //Constructor of Class
    Calculate(float no)
    {
        this.No = no;
    }
    //Behaviour of Class
    double CircleArea()
    {
        double area = 0.0f;

        area = 3.14 * this.No * this.No;

        return area;
    }
}

///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Main Class
class Area
{
    public static void main(String args[])
    {
        float No = 0.0f;
        double fret = 0.0f;

        System.out.println("Jay Ganesh...");

        //Creating object of Scanner Class To Take Input From User
        Scanner sobj = new Scanner(System.in);
        
        //Taking Input From user
        System.out.println("Enter Radius of Circle...");
        No = sobj.nextFloat();

        //Creating object of Calculate Class 
        Calculate cobj = new Calculate(No);
        //Call To Behaviour of Class
        fret = cobj.CircleArea();

        System.out.println("Area of Circle of is : "+fret);
    }
}

File: AreaOfCircle/Area.py
"""
    Author : Pakshal Jain
    Date : 28/10/2022
    Program : To Accept Radius From User and Calculate Area Of Circle
"""

####################################################################################################################################

#Class Declaration
class CircleArea :
    #Characteristics of Class
    No = 0.0;

    #Constructor of Class
    def __init__(self,no) :
        self.No = no;

    #Behaviour of Class
    def Calculate(self) :
        area = 0.0;

        area = 3.14 * self.No * self.No;

        return area;

#######################################################################################################################

#Main Function    
def main() :
    No = 0.0;
    fret = 0.0;

    print("Jay Ganesh...");
    
    #Taking Input From User
    print("Enter Radius of Circle");
    No = float(input());

    #Creating object of CircleArea Class
    cobj = CircleArea(No);
    #Call To Behaviour of Class
    fret = cobj.Calculate();

    print("Area of Circle is : ",fret);

##################################################################################################################################

#Call Back Method To Call Main Function
if __name__ == "__main__" :
    #Call To Main Function
    main();